bedrock/firefox/templates/firefox/features/index.html

```diff
@@ -79,61 +79,73 @@ <h2 class="section-title"><span>{{ _('Modern') }}</span></h2>
         </p>
       </li>
       <li class="features-list-item">
-        <a href="{{ url('firefox.android.index') }}" data-link-type="link" data-link-name="Firefox for Android" data-link-position="modern 5">
+        <a href="{{ url('firefox.features.send-tabs') }}" data-link-type="link" data-link-name="Tabs That Travel" data-link-position="modern 5">
```

Comment:
Actually, since these strings don't appear to be used in other templates, and don't need to put them in shared, we should be able to put them in firefox/features/index.lang and use a tag.